README.md

```diff
@@ -10,7 +10,7 @@
 
 ![Banner](docs/banner_image.png)
 
-This repo contains software for operating UUVs, developed by students at NTNU. The software is based on the ROS Melodic framework, and aims to be hardware independent. Although the main focus of Vortex is autonomous operation, this software stack supports both AUV and ROV operations.
+This repo contains software for operating UUVs, developed by students at NTNU. The software is based on the ROS2 Humble framework, and aims to be hardware independent. Although the main focus of Vortex is autonomous operation, this software stack supports both AUV and ROV operations. The goal is to achieve given tasks at the TAC Challenge fully autonomously. This repro supports therefore autonomous docking and autonomous pipline following. 
```

Comment:
*repo

On another note, the README here is probably deserving of a greater touch-up at some point! Adding some pictures of the Vortex AUVs etc etc ❤️ Could also be nice to have a section showing pipeline status of main for this repo and any dependencies we host ourselves

Reply:
You are totally right! This is just a quick fix. I already mentioned to Devops and some leads that we have to change the whole RADME at some point.